packages/web-components/src/components/va-process-list/va-process-list.css

```diff
@@ -0,0 +1,46 @@
+:host {
+  display: block;
+  list-style: none;
+  padding: 1em 0;
+  position: relative;
+}
+
+ol {
+  margin: 0 0 0 1.25em;
+  list-style-position: outside;
+}
+
+::slotted(li:first-child) {
+  counter-reset: listCounter;
+}
+
+::slotted(li) {
+  counter-increment: listCounter;
+  border-left: 8px solid var(--color-gray-light);
+  padding: 0 0 2em 2em;
+  list-style: none;
+  margin: 0 !important;
+}
+
+::slotted(li):before {
+  color: var(--color-white);
+  content: counter(listCounter);
+  float: left;
+  font-size: 1.3em;
+  font-weight: 700;
+  text-align: center;
+  width: 2em;
+  top: -0.2em;
+  margin-left: -2.7em;
+  display: block;
+  border: 4px solid var(--color-white);
+  background: var(--color-gray);
+  border-radius: 4em;
+  position: relative;
+  box-sizing: border-box !important;
+}
```

Comment:
General question do we want to use `rems` instead of `ems` for all of our `web-components` css?

Reply:
I don't have an answer for that :shrug:

Reply:
@k80bowman do you have an opinion on `ems` vs `rems` I know in some recent PRs we went `rem` and wanted to see if we wanted to set a standard for all of our web component css files

Reply:
I think it's a good idea to have a standard. I personally prefer `rem` for the design system because it scales with the document, rather than `em`, which is more variable because it can scale with the component. But I'm open to discussion on this if there are other viewpoints.

Reply:
I would personally prefer `rem` and looking at the files for Web Components css its like 70/30 rem vs em so I think we should stick with `rem` and create a ticket to convert any of the components that aren't using `rems` in their css to switch over to that

Reply:
I don't have a preference, either one is better than pixels as far as I'm concerned.